### PR TITLE
Correct links to example in README

### PR DESCRIPTION
**Pull request by khalidharun**
Correct links to example in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ A full example leveraging other community modules is contained in the [test/fixt
 2. Define the ALB module using the following in your [main.tf](test/fixtures/main.tf):
 ```
 module "alb" {
-  source              = "github.com/terraform-community-modules/tf_aws_alb//alb"
+  source              = "github.com/terraform-community-modules/tf_aws_alb/alb"
```
Comment (antonbabenko) [inline README.md]: There should be double slashes.

`Note: The double-slash, //, is important. It is what tells Terraform that that is the separator for a subdirectory, and not part of the repository itself.` from https://www.terraform.io/docs/modules/sources.html

Comment (khalidharun) [reply README.md]: Thanks for the quick reply.  I appreciate the feedback.

Comment (brandonjbjelland) [reply README.md]: @khalidharun if you revert this bit of the PR I'll be happy to merge it in. Thanks for the link update!

Comment (khalidharun) [reply README.md]: Done.  Thanks. 👍

**Pull request merged**